x-pack/plugins/security_solution/common/risk_engine/risk_score_calculation/request_schema.ts

```diff
@@ -6,7 +6,9 @@
  */
 
 import * as t from 'io-ts';
-import { DataViewId } from '../../api/detection_engine';
+// TODO https://github.com/elastic/security-team/issues/7491
+// eslint-disable-next-line no-restricted-imports
+import { DataViewId } from '../../api/detection_engine/model/rule_schema_legacy';
```

Comment:
Do you happen to know why this is a restricted import? They're both common files in the same plugin.

Reply:
The path is restricted because it includes `_legacy` in it. We have plans to remove those legacy schemas, and in the meantime, we don't want people to use them accidentally.